docs/object-functions.md

```diff
@@ -53,12 +53,27 @@ __Examples__
       "Postcode: SO21 2JN"
     ]
 
- ## `$error()`
+## `$error()`
 __Signature:__`$error(message)`
 
 Deliberately throws an error with an optional `message`
 
- ## `$assert()`
+## `$assert()`
 __Signature:__`$assert(condition, message)`
 
 If condition is true, the function returns undefined. If the condition is false, an exception is thrown with the message as the message of the exception.
+
+## `$typeOf()`
+__Signature:__`$typeOf(value)`
```

Comment:
I think I prefer the name `$type` as you proposed originally in #208.  In general I favour names as single words, provided the word is a meaningful representation of what the function does.  I think in this case `$type` would be ok?

Reply:
Sure.  Most languages (e.g. JavaScript) call the operator `typeof` so I went with that pattern.  If single word function names are preferred, we can use that.